modules/fts/pages/fts-query-types.adoc

```diff
@@ -346,6 +349,8 @@ More detailed information is provided in xref:query-string-queries.adoc[Query St
 _Term_ and _Phrase_ queries support no analysis on their inputs.
 This means that only exact matches are returned.
 
+Please note that xref:fts-query-types.adoc#prefix-query[Prefix], xref:fts-query-types.adoc#regexp-query[Regex] and xref:fts-query-types.adoc#wildcard-query[Wildcard] queries are also non-analytic queries.
```

Comment:
You've already pointed out earlier that prefix and wildcard queries are non-analytic. Why again here in the same page?

Reply:
People reading about non-analytic queries should get a list of such ones FTS supports. Hence felt that it is important to have a single place where it list all of them